src/librustc_typeck/check/coercion.rs

```diff
@@ -108,6 +108,7 @@ fn coerce_mutbls<'tcx>(
     }
 }
 
+/// Do not require any adjustments, i.e. coerce `x -> x`.
```

Comment:
cc @nikomatsakis (comments are great but I'm not sure of the accuracy)

Reply:
This comment seems correct, anyway